Containers/Profile.js

```diff
@@ -151,7 +192,8 @@ const styles = {
 
 const mapStateToProps = state => {
   return {
-    favorites: state.user.favorites,
+    favorite_garments: state.user.favorite_garments,
```

Comment:
sometimes we use favorite_garments an sometimes we use favoriteGarments. Aside from favorite_garments coming from the backend, does the front end ever use it?

Reply:
favorite_garments comes from the backend response which is being stored using redux. Calling this.props.favorite_garments is pulling from that store. I kept the name initially, to be consistent with the api. One of the benefits is being able to transfer and access application state easily from different components.

this.props.favorite_garments is different from this.state.favoriteGarments. One is coming from the redux store, and the other is an internal component level state.

I can definitely map to camelCase variables in UserRedux if you feel that is cleaner.